### PR TITLE
Changed all usage of Base64 to Base64Url on fe1-web

### PR DESCRIPTION
**Pull request by Adalsteinnjons**
All Ids and data encodings are now in base64Url instead of normal base64.
This is linked to issue #281.

### CODE REVIEW

**Review by RaulinN — DISMISSED** [review]

**Review by pierluca — DISMISSED** [review]

**Comment by Adalsteinnjons** [comment]
Tested with @flmichel base64 url branch from the backend and it works.

File: fe1-web/model/objects/Base64Url.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  public toBuffer(): Buffer {
+    return base64url.toBuffer(this.valueOf());
+  }
+
+  public static fromBase64(b64: string): string {
+    return this.addPadding(base64url.fromBase64(b64));
+  }
+
+  public static addPadding(str: string): string {
```
Comment (pierluca) [inline fe1-web/model/objects/Base64Url.ts:31]: this looks like it probably shouldn't be public.

File: fe1-web/model/objects/Base64Url.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  public equals(o: Base64UrlData): boolean {
+    return this.toString() === o.toString();
+  }
+
+  public toBuffer(): Buffer {
+    return base64url.toBuffer(this.valueOf());
+  }
+
+  public static fromBase64(b64: string): string {
+    return this.addPadding(base64url.fromBase64(b64));
```
Comment (pierluca) [inline fe1-web/model/objects/Base64Url.ts:28]: shouldn't this return a `Base64UrlData` ?

**Review by pierluca — CHANGES_REQUESTED** [review]
A couple of minor points need addressing, but it seems pretty good overalls 👍

File: fe1-web/model/objects/Base64Url.ts (hunk 1 above)
Comment (Adalsteinnjons) [reply fe1-web/model/objects/Base64Url.ts:31]: Correct

File: fe1-web/model/objects/Base64Url.ts (hunk 2 above)
Comment (Adalsteinnjons) [reply fe1-web/model/objects/Base64Url.ts:28]: Correct

**Review by pierluca — APPROVED** [review]
LGTM

**Pull request merged**